app/controllers/users.js

```diff
@@ -0,0 +1,48 @@
+const emailDomain = '@wolox.com.ar',
+  bcrypt = require('bcrypt'),
+  { databaseError, hashError, validationError } = require('../errors'),
+  Users = require('../models').user,
+  logger = require('../logger'),
+  saltRounds = 10,
+  minPasswordLength = 8,
+  alphanumericRegex = /^[0-9a-zA-Z]+$/;
+
+// completar estas funciones
```

Comment:
Remove this comment in the next PR. Try not to leave TODOs. Comments explaining complex code are fine though, but they have to be in english.